lib/views/help/api.es.html.erb

```diff
@@ -44,6 +44,7 @@
 
       <p>Solicitudes, usuarios y autoridades, todos tienen una versión JSON con informaciones básicas sobre ellos. Cada formato Atom tiene un equivalente JSON con informaciones sobre los objetos incluidos en el feed.</p>
 
+    <h2>Información sobre todos los entidades públicas de España</h2>
```

Comment:
This should read Información sobre todos los entidades públicas de Nicaragua not España :)